src/components/CytoscapeLayout/CytoscapeLayout.tsx

```diff
@@ -43,9 +44,24 @@ export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProp
         const elements: { [key: string]: any } = response['data'];
         console.log(elements);
         this.setState(elements);
+
+        let canvasWrapper = document.getElementById('cytoscape-container')!;
```

Comment:
We probably need to eliminate these `getElementById` altogether as they stop reusabilty. We can't reuse these components in any other pages then.
I know there was already one instance of this introduced by @cfcosta.

Reply:
That explains why the test smoke test blew up.  We should re-enabled the [test](https://github.com/swift-sunshine/swsui/pull/75) when addressing `document.getElementById()` 
 
https://travis-ci.org/swift-sunshine/swsui/builds/347950574
```
src/pages/ServiceDetails/ServiceMetrics.tsxsrc/pages/ServiceDetails/ServiceMetrics.tsx 137ms
src/pages/ServiceGraph/ServiceGraphPage.tsxsrc/pages/ServiceGraph/ServiceGraphPage.tsx 15ms
src/pages/ServiceList/ServiceListComponent.tsxsrc/pages/ServiceList/ServiceListComponent.tsx 136ms
src/pages/ServiceList/ServiceListPage.tsxsrc/pages/ServiceList/ServiceListPage.tsx 33ms
src/registerServiceWorker.jssrc/registerServiceWorker.js 45ms
src/registerServiceWorker.tssrc/registerServiceWorker.ts 55ms
src/services/Api.tssrc/services/Api.ts 34ms
src/setupTests.tssrc/setupTests.ts 4ms
src/types/Metrics.tssrc/types/Metrics.ts 4ms
src/types/Namespace.tssrc/types/Namespace.ts 2ms
src/types/NamespaceId.tssrc/types/NamespaceId.ts 2ms
src/types/ServiceFilter.tssrc/types/ServiceFilter.ts 10ms
src/types/ServiceId.tssrc/types/ServiceId.ts 3ms
src/types/ServiceInfo.tssrc/types/ServiceInfo.ts 20ms
src/types/ServiceListComponent.tssrc/types/ServiceListComponent.ts 6ms
/home/travis/build/swift-sunshine/swsui/node_modules/react-scripts-ts/scripts/test.js:20
  throw err;
  ^
TypeError: Cannot read property 'getElementById' of undefined
    at /home/travis/build/swift-sunshine/swsui/src/components/CytoscapeLayout/CytoscapeLayout.tsx:53:95
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
``